leaflet-geoman.d.ts

```diff
@@ -954,16 +969,42 @@ declare module 'leaflet' {
 
 
         interface PMLayer extends PMRotateLayer, PMEditLayer, PMDragLayer {
-            /** get shape */
+            /** get shape of the layer. */
             getShape(): SUPPORTED_SHAPES;
         }
 
+        interface PMLayerGroup {
+            /** Enables edit mode for all child layers. The passed options are preserved, even when the mode is enabled via the Toolbar */
+            enable(options?: EditModeOptions): void;
```

Comment:
@ryan-morris can you please check if this works, because the `enable` function of the LayerGroup has a private argument `_layerIds`. The user should only use the `options` argument
https://github.com/geoman-io/leaflet-geoman/blob/06c41aa1e8db8ea89f251eb25e4ad554d363250d/src/js/Edit/L.PM.Edit.LayerGroup.js#L54

Reply:
the typings for this just don't expose _layerIds as a parameter at all. So all the user should see as available is the options argument

Reply:
Yes exactly